test/bdd/pkg/kms/auth_edv_steps.go

```diff
@@ -7,40 +7,19 @@ SPDX-License-Identifier: Apache-2.0
 package kms
 
 import (
-	"encoding/base64"
 	"fmt"
-	"net/http"
-	"strings"
-
+	kmsapi "github.com/hyperledger/aries-framework-go/pkg/kms"
 	"github.com/hyperledger/aries-framework-go/pkg/vdr/fingerprint"
-	"github.com/lafriks/go-shamir"
-	"github.com/rs/xid"
-	"github.com/trustbloc/edge-core/pkg/zcapld"
-	"github.com/trustbloc/edv/pkg/client"
-	"github.com/trustbloc/edv/pkg/restapi/models"
 
 	"github.com/trustbloc/kms/test/bdd/pkg/auth"
-	"github.com/trustbloc/kms/test/bdd/pkg/internal/cryptoutil"
-)
-
-const (
-	edvBasePath    = "/encrypted-data-vaults"
-	secretEndpoint = "/secret"
 )
 
-func (s *Steps) storeSecretInHubAuth(userName string) error {
+func (s *Steps) loginIntoAuthServer(userName string) error {
```

Comment:
Typo here - I think you mean `logInToAuthServer`

Reply:
fixed